requirements.txt

```diff
@@ -0,0 +1,14 @@
+yaxil>=0.5.2
+paramiko>=2.7.2
+boxsdk>=2.11.0
+jsonpath_ng>=1.5.2
+cryptease>=0.2.0
+pytz>=2021.1
+requests>=2.26.0
+six>=1.16.0
+pandas>=1.3.2
+pytest>=6.2.4
+numpy>=1.20.3
+mano>=0.5.1
+LAMP>=0.0.1
+PyYAML>=6.0
```

Comment:
I see that this file is a copy of the packages in `setup.py`. Why is this one required then? Regardless, updating packages in two places will be burdensome in future. This is how it is done for convenience:

https://github.com/pnlbwh/conversion/blob/48f94ce21005d1af779f98df0c5ea8ec0e204835/setup.py#L9-L18

```python
requirements= open('requirements.txt').read().split()

setup(name='conversion',
    version=__version__,
    ...
    install_requires=requirements,

```

I can help with it.

Reply:
@kcho 

> Why is this one required then?

Reply:
Based on my discussion with Kevin, `requirements.txt` is redundant and so it should be removed.